### ISSUE TITLE
RNN-T recipe for LibriSpeech

### ISSUE THREAD

**Issue by tu1258** [espnet/espnet#1704]
Based on test by #1248. I modified `run_rnnt.sh` from voxforge and try to run on LibriSpeech, but I then got permission denied. 
[My version rnnt.sh](https://github.com/espnet/espnet/files/4356328/rnnt_librispeech.txt)
Actually I think my version may not be correct to reproduce same performance as #1248. Is there anyone sharing about RNN-T script for LibriSpeech? If no, I wish someone will do since LibriSpeech is an important benchmark in ASR. Thanks!

**Comment by b-flo** [comment]
Hi,

I suppose default run script (https://github.com/espnet/espnet/blob/master/egs/librispeech/asr1/run.sh) was used in #1248, with `recog_model=model.loss.best` and the config(s) shared in the post. Did you try? Is there any degradation compared to reported results?

**Comment by tu1258** [comment]
The scripts are so different. What should I set `nbpe` and `bpemode` if I use grapheme model

**Comment by b-flo** [comment]
Sorry, I think there is some misunderstanding. 

- To use transducer, you only need to set the following in train config:

```
model-module: "espnet.nets.pytorch_backend.e2e_asr_transducer:E2E"
criterion: loss
mtlalpha: 1.0
```

- `run_rnnt.sh` was originally created to demonstrate rnn-t and finetuning, however it is recipe-dependant and was not intended as a general template like `run.sh`. 
It is just an "all-in-one" training script with some control parameters at bash level, that's why it's so different. You can achieve the same thing by training one or multiple times `run.sh` by the way. For example, `run_rnnt.sh` without transfer learning enabled is equivalent to using `run.sh`.

**Comment by tu1258** [comment]
OK, I'll try it. Thanks!

**Comment by tu1258** [comment]
Hi, @b-flo 
I tried it and get `basename: missing operand` error at Dictionary and Json Data Preparation

**Comment by b-flo** [comment]
Looking at librispeech run script, the only `basename` would could produce this error at this stage (well it's between stage 2 and 3) is this one:
```
if [ -z ${lmtag} ]; then                                                                                                                                                                                     
    lmtag=$(basename ${lm_config%.*})                                                                                                                                                                        
fi  
```
`lm_tag` is not set by default, so except if you set the value and `lm_config` was empty, I'm not sure what happened. Are you using librispeech run script with only `recog_model=model.loss.best` and configs modified?

**Comment by tu1258** [comment]
Sorry my bad. I thought RNN-T don't need LM and set it empty. Will try it!
